test/draw_line_string.test.js

```diff
@@ -137,7 +137,7 @@ test('draw_line_string render active line with 1 coordinate', t => {
     },
     geometry: {
       type: 'LineString',
-      coordinates: [[0, 0]]
+      coordinates: [[0, 0], [15, 15]]
```

Comment:
If it was failing I think that's because of your change at https://github.com/mapbox/mapbox-gl-draw/pull/453/files#diff-7a0124268c46d39459f7595fd9d4d1caR83. The test was correct for the prior behavior, but you're changing that, right?